cmd/kubelet/app/options/options.go

```diff
@@ -397,6 +397,7 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.Var(cliflag.NewColonSeparatedMultimapStringString(&c.StaticPodURLHeader), "manifest-url-header", "Comma-separated list of HTTP headers to use when accessing the url provided to --manifest-url. Multiple headers with the same name will be added in the same order provided. This flag can be repeatedly invoked. For example: --manifest-url-header 'a:hello,b:again,c:world' --manifest-url-header 'b:beautiful'")
 	fs.Var(&utilflag.IPVar{Val: &c.Address}, "address", "The IP address for the Kubelet to serve on (set to '0.0.0.0' or '::' for listening in all interfaces and IP families)")
 	fs.Int32Var(&c.Port, "port", c.Port, "The port for the Kubelet to serve on.")
+	fs.Var(&utilflag.IPVar{Val: &c.ReadOnlyBindAddress}, "read-only-bind-address", "The address used for the read-only API on, when enabled by setting --read-only-port. Defaults to the value passed to --address. Has no effect if ReadOnlyPort is disabled. (set to '0.0.0.0' or '::' for listening in all interfaces and IP families)")
```

Comment:
Yes, we do want to get rid of it. However, AIUI, there is still [some work] to be done before we remove the code completely.

[some work]: https://github.com/kubernetes/kubernetes/issues/12968#issuecomment-622613486

This is a relatively cheap change that, IMO, 
1. helps reduce the risks of having to enable the read-only port,
2. does not substantially increase the amount of work to actually remove ro-port (we will be removing two flag and config items instead of one, in the same place).